data-pipeline-ffi/src/trace_exporter.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright 2024-Present Datadog, Inc. https://www.datadoghq.com/
+// SPDX-License-Identifier: Apache-2.0
+
+use bytes::Bytes;
+use data_pipeline::trace_exporter::TraceExporter;
+use data_pipeline::trace_exporter::TraceExporterBuilder;
+use ddcommon_ffi::{
+    slice::{AsBytes, ByteSlice},
+    CharSlice,
+};
+use std::ffi::{c_char, CString};
+
+#[no_mangle]
+pub unsafe extern "C" fn dd_trace_exporter_new(
```

Comment:
Was there a discussion anywhere about `dd_trace`, `ddtrace`, etc? We've generally been using `ddog_` as a prefix to avoid possible conflicts in client libraries (which have been around a lot longer) and may be using those `dd_trace`, `ddtrace` style prefixes.

It's not a blocker, just curious.

Reply:
No discussion at all :), just pick that name up to start coding. I'm gonna change it to be consistent with the rest of the crates. Thanks for the suggestion.